pkg/convert/wolfios/wolfi-os.go

```diff
@@ -0,0 +1,148 @@
+package wolfios
```

Comment:
Can we break this package out from under convert/apkbuild like we did with manifest and http?

Reply:
done